base/changes.txt

```diff
@@ -6,6 +6,12 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2024-04-17 Joseph Wright  <Joseph.Wright@latex-project.org>
+	* ltexpl.dtx, ltdefns.dtx
+	Rename \@expl@cs@argument@spec@@N to \@expl@cs@parameter@spec@@N (gh/1014)
+	* ltexpl.dtx, ltcmd.dtx, ltcmdhooks.dtx
+	Use \cs_parameter_spec:N in place of \cs_argument_spec:N (gh/1014)
```

Comment:
Mention `\__kernel_cs_parameter_spec:N` too?

Reply:
I don't think the changes needs to go to that level of detail myself

Reply:
We don't normally roll back internals unless there is a very good reason. Being internal they should not confliced with anything if they stay around.

Reply:
ups comment in the wrong place :-)